<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSettings</class>
 <widget class="QDialog" name="ProjectSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>Общее</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="classes">
      <attribute name="title">
       <string>Классы</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>681</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnSortAZ">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../qlabelme.qrc">
              <normaloff>:/images/resources/sort.svg</normaloff>:/images/resources/sort.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDown">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../qlabelme.qrc">
              <normaloff>:/images/resources/down_arrow.svg</normaloff>:/images/resources/down_arrow.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUp">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../qlabelme.qrc">
              <normaloff>:/images/resources/up_arrow.svg</normaloff>:/images/resources/up_arrow.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEdit">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../qlabelme.qrc">
              <normaloff>:/images/resources/edit.svg</normaloff>:/images/resources/edit.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnAdd">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../qlabelme.qrc">
                <normaloff>:/images/resources/add.svg</normaloff>:/images/resources/add.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRemove">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../qlabelme.qrc">
                <normaloff>:/images/resources/delete.svg</normaloff>:/images/resources/delete.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listClasses"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qlabelme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
